tests/block/k8s/integration/k8sBlockStorageIntegration_test.go

```diff
@@ -35,6 +36,11 @@ import (
 // Rook Block Storage integration test
 // Start MySql database that is using rook provisoned block storage.
 // Make sure database is functional
+var (
+	// Env var containing environment manifest
+	Env  objects.EnvironmentManifest
+	rook *installer.InstallHelper
```

Comment:
could the `Env` be put inside the InstallHelper? Then there is only one context object